client/domain/CodeEvalResultObjectFactorySpec.js

```diff
@@ -21,8 +21,8 @@ describe('CodeEvalResultObjectFactory', function() {
   var codeEvalResult;
   var CODE = "code";
   var OUTPUT = "output";
-  var CORRECTNESS_TEST_RESULTS = "correctnessTestResults";
-  var LAST_TASK_RESULTS = "s";
+  var CORRECTNESS_TEST_RESULTS = ["xyz", "xyz", "xyz", "xyz", "xyz"];
+  var LAST_TASK_RESULTS = "xyz";
   var BUGGY_OUTPUT_TEST_RESULTS = "buggyOutputTestResults";
```

Comment:
Want to change this too (and the line below)? Try and figure out the expected format by console.logging / examining the calling code.